### DISCUSSION TITLE
tpscript is bad

### DISCUSSION THREAD

**Discussion by raymonable** [headsmasher8557/tpscript#9] [Show and tell]
luaasm is also bad
but tpscript is worse

**Comment by ghost** [comment]
yes tpscript is very dangerous and bad you can use loadstring to run very dangerous code and you can make you codebase explode using this danger secure vulnerable i am so angry at this issue it can destroy my codebase if someone uses it and i dont like it

**Comment by headsmasher8557** [comment]
luau user.... smh....
